examples/group_provisioning_demo.c

```
@@ -0,0 +1,275 @@
+
+    printf("\n[Leader] Sending encrypted GROUP_JOIN invite to Member A (%04X)...\n", MEMBER_A);
+    {
+        /* Build a GROUP_JOIN payload: this is what would be sent over the air,
+         * normally encrypted under the pairwise session key with Member A. */
+        group_join_payload_t join;
+        join.group_id = GROUP_ID_1;
+        join.epoch    = 1;
+        memcpy(join.group_key, group_key, SYMMETRIC_KEY_LENGTH);
+
+        packet_t pkt;
+        if (packet_create(&pkt, MEMBER_A, PACKET_TYPE_GROUP_JOIN,
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)